doc/user-stories/expert-partitioner.md

```diff
@@ -0,0 +1,414 @@
+    * shows option 'Logical partition'
+
+### When we are in wizard step to select the partition size
+* shows three options
+  * Maximun size
+  * Custom size
+    * with maximum size by default
+  * Custom region
+    * with start and end cylinders of the free region by default
+* *(pending) and partition is primary*
+  * custom size is selected by default
+* and partition is extended
+  * maximum size is selected by default
+* *(pending) and partition is logical*
+  * custom region is selected by default
+* and 'Next' is selected
+  * and partition is extended
+    * finishes wizard
+    * creates the partition
+
+### When we are in wizard step to select the role of the partition
+* shows the following options
+  * operating system (i.e. root)
+  * data and ISV applications (any other mount point)
+  * swap
+  * raw volume (unformatted)
+* option 'data and ISV applications' is selected by default
+
+### When we are in wizard step for partition attributes
+* shows two sets of options: 'Formatting options' and 'Mounting options'
+  * where 'Formatting options' contains
+    * Format partition
+    * Do not format partition
+    * Encrypt device
+  * where 'Mounting options' contains
+    * Mount partition
+    * Do not mount partition
+* *(pending) and partition role is 'Operating system'*
+  * sets the following 'Formatting options' by default
+    * Format partition: true
+      * File System: Btrfs
+      * Enable snapshots: true
+    * Do not format partition: false
+      * File System Id: 0x83 Linux
+    * Encrypt: false
+  * sets the following options for 'Mounting options'
+    * Mount partition: true
+      * Mount point: first free of /, /home, /var
+    * Do not mount partition: false
+* *(pending) and partition role is 'Data and ISV applications'*
+  * sets the following 'Formatting options' by default
+    * Format partition: true
+      * File System: XFS
+    * Do not format partition: false
+      * File System Id: 0x83 Linux
+    * Encrypt: false
+  * sets the following options for 'Mounting options'
+    * Mount partition: true
+      * Mount point: first free of /, /home, /var
+    * Do not mount partition: false
+* *(pending) and partition role is 'Swap'*
+  * sets the following 'Formatting options' by default
+    * Format partition: true
+      * File System: Swap
+    * Do not format partition: false
+      * File System Id: 0x82 Linux Swap
+    * Encrypt: false
+  * sets the following options for 'Mounting options'
+    * Mount partition: true
+      * Mount point: swap
+    * Do not mount partition: false
+* *(pending) and partition role is 'Raw volume'*
+  * sets the following 'Formatting options' by default
+    * Format partition: false
+    * Do not format partition: true
+      * File System Id: 0x8E Linux LVM
+    * Encrypt: false
+  * sets the following options for 'Mounting options'
+    * Mount partition: false
+    * Do not mount partition: true
+* *(pending) and 'Finish' button is selected*
```

Comment:
I don't think so. First, validations are not done and it seems that selected values are not saved, or at least they are not correctly loaded when then you edit.